homeassistant/components/homewizard/switch.py

```diff
@@ -30,6 +30,13 @@ async def async_setup_entry(
             ]
         )
 
+    if coordinator.data["system"]:
+        async_add_entities(
```

Comment:
It's nice to collect all possible entities in a list first and then make a single call to `async_add_entities`.